src/sign/ed25519.rs

```diff
@@ -263,7 +263,9 @@ pub mod thrussh {
             buf.extend_ssh_string(pk.as_ref());
             buf.push_u32_be(64);
             buf.extend(&pair);
-            buf.extend_ssh_string(b"");
+            // The GnuPG SSH agent fails to add keys with empty comments.
+            // See: https://dev.gnupg.org/T5794
+            buf.extend_ssh_string(b"radicle");
```

Comment:
I'm wondering if we should differentiate which type of key we're using, so something like:
```
"radicle ed25519-zebra"
```
just for traceability for bug reports, etc.

Reply:
Sounds good